File: templates/service-java/{{#server.spring}}/src/main/java/{{package.value}}/{{mainclass.value}}.java
package {{package.value}};

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
{{#swagger.value}}
import springfox.documentation.swagger2.annotations.EnableSwagger2;
{{/swagger.value}}

{{#swagger.value}}
@EnableSwagger2
{{/swagger.value}}
@SpringBootApplication
public class {{mainclass.value}} {

	public static void main(String[] args) {
		SpringApplication.run({{mainclass.value}}.class, args);
	}
}

File: templates/service-kotlin/{{#server.spring}}/src/main/kotlin/{{package.value}}/{{mainclass.value}}.kt
package {{package.value}}

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
{{#swagger.value}}
import springfox.documentation.swagger2.annotations.EnableSwagger2
{{/swagger.value}}

{{#swagger.value}}
@EnableSwagger2
{{/swagger.value}}
@SpringBootApplication
class {{mainclass.value}}

fun main(args: Array<String>) {
    runApplication<{{mainclass.value}}>(*args)
}
